### PR TITLE
feat: xref graph ci

### PR DESCRIPTION
**Pull request by kyle-annen**
Adds xref graph generation and comment on successful PR run.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -125,3 +126,25 @@ jobs:
 
       - name: Elm Tests
         run: npm run elm-test --prefix assets
+
+      - name: Xref Graph Gen/Upload
+        id: xref
+        run: |
+          mix xref graph --format dot
+          sudo apt-get install -y jq graphviz
```
Comment (7hoenix) [inline .github/workflows/ci.yml:134]: It probably doesn't matter, but I wonder if this should be up with the other installation bits.

Comment (7hoenix) [reply .github/workflows/ci.yml:134]: Not sure if it is like docker and could realize that it hasn't changed in that layer (and then not execute it again).

Comment (kyle-annen) [reply .github/workflows/ci.yml:134]: I was trying to get it to fail as fast as possible, so deferring installation steps until needed. Though it is probably a bunch cleaner if I do it in one spot.

**Comment by 7hoenix** [comment]
This is great! 🎉  :shipit:

File: .github/workflows/ci.yml (hunk 1 above)
Comment (kyle-annen) [reply .github/workflows/ci.yml:134]: Will refactor in the next PR, will still be in this file.

**Pull request merged**